### DISCUSSION TITLE
How can edit/customize a default theme

### DISCUSSION THREAD

**Discussion by k-msalehi** [Laravel-Backpack/community-forum#775] [Q&A (Help)] [answered]
I've read the [doc about how to create a theme](https://backpackforlaravel.com/docs/6.x/add-ons-tutorial-how-to-create-a-theme).
I've tried to copy CoreUIv4 Theme from vendor/backpack and move it to my resource/views, but I can't figure how to use it as seperate theme.
I need to replace some of default js and css files (replace, not add), and change some class names in template.
How can I edite CoreUIv4 theme?

**Comment by karandatwani92** [comment]
Hey @msalehi-d 

Here's an article to help you with this
https://backpackforlaravel.com/articles/tutorials/new-in-v6-how-to-create-your-own-custom-theme-to-backpack-v6

Let me know, if I can help you with something else.

Happy Coding!

**Comment by k-msalehi** [comment]
@karandatwani92, thanks for that link, I tried that before, but there are some problems.
for example, when I check source code of admin panel in browser, it has `dataTables.bootstrap5.min.css`  there is a file in `/vendor/backpack/crud/src/resources/views/crud/list.blade.php` that calls this file and some other CSS and JS, I need to modify it, but I with that solution, I can't, unless I edit that file directly in the vendor directory, which it not good practice.

**Reply by karandatwani92** [reply, marked as answer]
To customize list operations view.

copy view `vendor/backpack/crud/src/resources/views/crud/list.blade.php`

paste here `resources/views/vendor/backpack/crud/list.blade.php` and change it as you like
